RELEASE.txt

```diff
@@ -4,6 +4,27 @@ RELEASE=2.3
 
 # Release History
 
```

Comment:
Another thing you'll see is that instantiating TestRun objects now has two steps. 

1. Creating the object `TestRun()` - Initializes the object from its config, and creates a test run dir.
2. Saving the object `test_run.save()` - Creates all the files related to the test run and saves them. 

This will become important in future merge requests, as it separates out almost all the IO from getting the test to a state where we can evaluate if it should be skipped.